package.json

```diff
@@ -32,6 +32,9 @@
     "webpack": "^4.8.3",
     "webpack-cli": "^2.1.3"
   },
+  "peerDependencies": {
```

Comment:
What's the reason for adding it to peerDependencies instead of dependencies?

Reply:
Because chalk need only for devDependencies in host. So user can decide by self when it should located

Reply:
Sorry, can't follow. `asinit`, which is essentially a production entity, makes use of it, so shouldn't it be installed normally so a user can start using `asinit` right away without any further steps?

Reply:
Ok, I'll move this to normal deps